assets/i18n/qqq.json

```diff
@@ -556,6 +556,7 @@
 	"I18N_TOPNAV_BLOG": "Text displayed inside a drop-down menu. - The list is shown after the user hovers over the I18N_TOPNAV_ABOUT button. When the option is clicked, the Oppia blog is loaded.\n{{Identical|Blog}}",
 	"I18N_TOPNAV_CLASSROOM": "Text displayed inside the navigation bar. - When the user hovers over the button, a list with several classroom pages is displayed.\n{{Identical|Classroom}}",
 	"I18N_TOPNAV_CLASSROOM_BASIC_MATHS": "Text displayed inside drop-down menu. - The list is shown after the user hovers over the I18N_TOPNAV_CLASSROOM button. When the option is clicked, the Maths Classroom page is loaded.",
+	"I18N_TOPNAV_COMMUNITY_DASHBOARD": "Text displayed inside a drop-down menu. - The list is shown after the user hovers his profile picture. When the option is clicked, the community dashboard page is loaded where the user can contribute translation and questions for lessons.",
```

Comment:
his --> over their
loaded where the --> loaded. On this page, the

Reply:
Done!